css/index.css

```diff
@@ -79,6 +79,46 @@ p {
     border-bottom: 1px dashed black;
 }
 
+.about-page .top-content {
+    display: block;
+    padding: 2% 2%;
+    margin-bottom: 30px;
+    border-bottom: 1px dashed black;
+}
+
+.about-page .middle-content {
+    display: flex;
+    flex-wrap: wrap;
```

Comment:
Is this necessary? Since the web page is not responsive (yet), this line isn't needed, but you may end up using it after introducing media queries.